shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -541,12 +541,14 @@ - (void)applicationBecameActive:(NSNotification*)notification {
   if (_viewportMetrics.physical_width)
     [self surfaceUpdated:YES];
   [self goToApplicationLifecycle:@"AppLifecycleState.resumed"];
+  [_engine.get() setIsGpuDisabled:NO];
```

Comment:
Can we just make the engine subscribe to these notifications instead, so this will work even if there's no VC?

Reply:
(FlutterEngine that is)

Reply:
Sure, good idea, one sec.  The only problem is that ordering isn't deterministic between notifications.  I don't think that will be an issue here like it was on my other PR.

Reply:
Done.